model.go

```diff
@@ -64,6 +64,11 @@ type Person struct {
 	Description string
 }
 
+const (
+	TemplateFull    = "full"
+	TemplateOnePage = "one-page"
+)
+
```

Comment:
Is it appropriate to put constants in `model.go`?

This was my first time touching go, but it seemed like a reasonable place to put them.